controls/radmulticolumncombobox/key-properties-and-methods.md

```diff
@@ -12,14 +12,14 @@ position: 3
 
 The article lists the most important properties and methods of __RadMultiColumnComboBox__.
 
-* [Delaying the Text Change](#delay-the-text-change)
+* [Text Change Without Delay](#text-change-without-delay)
 * [Disable Items Highlighting](#disable-items-highlighting)
 * [Setting Text for an Empty Selection](#setting-text-for-an-empty-selection)
-* [Programmatically Closing the DropDown](#programmatically-closing-the-dropdown)
+* [Open,Close and Toggle the DropDown](#open,close-and-toggle-the-dropdown)
 
-## Delaying the Text Change
+## Text Change Without Delay
 
-The default behavior of __RadMultiColumnComboBox__ for reacting to a text change will be triggered immediately. In case when a delay of the update is needed, the __DeferTextChange__ property can be set to __True__.
+By default, the reaction of __RadMultiColumnComboBox__ to a text change will be deferred. In case when updating needs to be done immediately, the __DeferTextChange__ property can be set to __False__.

```

Comment:
I don't understand this. What does it mean that the reaction will be deferred? Is it that the text will be changed on lost focus? Or is it something else. Can you describe this?